pkg/telemetry/telemetry_test.go

```diff
@@ -32,29 +32,33 @@ func TestSendTelemetry(t *testing.T) {
 				a.AddManaged(&resource.FakeResource{})
 				a.AddUnmanaged(&resource.FakeResource{})
 				a.Duration = 123.4 * 1e9 // 123.4 seconds
+				a.IgnoreRulesCount = 24
 				return a
 			}(),
 			expectedBody: &telemetry{
-				Version:        version.Current(),
-				Os:             runtime.GOOS,
-				Arch:           runtime.GOARCH,
-				TotalResources: 2,
-				TotalManaged:   1,
-				Duration:       123,
+				Version:          version.Current(),
+				Os:               runtime.GOOS,
+				Arch:             runtime.GOARCH,
+				TotalResources:   2,
+				TotalManaged:     1,
+				Duration:         123,
+				IgnoreRulesCount: 24,
 			},
 		},
 		{
 			name: "valid analysis with round up",
 			analysis: func() *analyser.Analysis {
 				a := &analyser.Analysis{}
 				a.Duration = 123.5 * 1e9 // 123.5 seconds
+				a.IgnoreRulesCount = 0
```

Comment:
You can remove this I think as this is not the prupose of this test case. I'd rather prefer to have minimal test cas with a single responsibility